timdex_dataset_api/dataset.py

```diff
@@ -30,15 +29,16 @@
         pa.field("run_type", pa.string()),
         pa.field("run_id", pa.string()),
         pa.field("action", pa.string()),
+        pa.field("year", pa.string()),
+        pa.field("month", pa.string()),
+        pa.field("day", pa.string()),
```

Comment:
The partition columns are set as `pa.string()` objects to support zero-padded months and dates.